### PR TITLE
Cw youre a square

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

**Review by W-Ely — DISMISSED** [review]
Probably this is meant to check for _perfect_ squares.

You have three code paths and only 2 tests.  What would the last test look like?

**Comment by aepaysinger** [comment]
it would be for less than 0

File: code_challenges/code_wars/youre_a_square.py [hunk 1]
```diff
@@ -14,4 +14,6 @@ def is_square(n):
 if __name__ == "__main__":
     n = 34
     print(is_square(n))
-    
\ No newline at end of file
+    
+
+    # fun things
```
Comment (W-Ely) [inline code_challenges/code_wars/youre_a_square.py]: Stray comment

**Review by W-Ely — CHANGES_REQUESTED** [review]
I didn't read the whole commit.  I see now it is a very long message.

**Comment by aepaysinger** [comment]
i think this is cleaned up now

**Review by W-Ely — APPROVED** [review]

**Pull request merged**